run.py

```diff
@@ -1165,11 +1166,12 @@ def main():
         if not args.patterns:
             tests.append(dirpath)
         else:
+            test_name = dirpath[len(tdir) + 1:]
             for pattern in args.patterns:
                 if args.exact:
-                    if pattern == basename:
+                    if pattern == test_name:
```

Comment:
So, now I need to say `dns/dns-rcode` instead of `dns-rcode` ?

Reply:
How so?
```
[jason@desktop …/dev/suricata/master]  master ❯ ~/oisf/dev/suricata-verify/master/run.py dns-frames
    Finished release [optimized] target(s) in 0.01s
Number of concurrent jobs: 8
===> dns/dns-frames: OK

PASSED:  1
FAILED:  0
SKIPPED: 0
```

Reply:
You did not use `args.exact` here

Reply:
Also This line does not fit with the commit message :-p

Reply:
> Does anyone actually use this feature?

I do not.

I guess this line change deserves its own commit, so that the chosen behavior is documented...